charts/k6-loadtester/templates/deployment.yaml

```diff
@@ -24,6 +24,11 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      volumes:
+        {{- toYaml .Values.volumes | nindent 8 }}        
+      initContainers:
+{{ toYaml .Values.extraInitContainers | indent 8 }}
```

Comment:
Indent seems wrong. Would look nicer with nindent like the other attributes. Also the value name is not `extraInitContainers`
```suggestion
{{- toYaml .Values.initContainers | nindent 8 }}
```